### PR TITLE
Add Fleet scaling recommendation: ES API key cache size

### PR DESCRIPTION
**Pull request by kilfoyle**
This updates the Fleet Server scalability page with a scaling tip running a large volume of agents.

Closes: #1156 

---

Screenshot 2024-07-09 at 12 46 15 PM

---

I also put a link at the top of the page to make sure people can find the scaling recommendations section.

Screenshot 2024-07-09 at 12 48 43 PM

### CODE REVIEW

File: docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc [hunk 1]
```diff
@@ -226,7 +228,19 @@ of these operations.
...
-== Policy scaling recommendations
+== Scaling recommendations
+
+**{agent} policies**
 
 A single instance of {fleet} supports a maximum of 500 {agent} policies. If more policies are configured, UI performance might be impacted.
 
+**{agents}**
+
+When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to `max number of agents * 2`.
```
Comment (shainaraskas) [inline docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc:239]: when I read this through the first time I got kind of confused. in retrospect the asterisk might be ok but here are some alternatives I already typed out 


```suggestion
When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to `max number of agents x 2`.
```

or

```suggestion
When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to `2(max number of agents)`.
```

or 

```suggestion
When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to 2x the maximum number of agents.
```

File: docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc [hunk 2]
```diff
@@ -226,7 +228,19 @@ of these operations.
...
+
+For example, with 25,000 running {agents} you could set the cache value to `50,000`:
```
Comment (shainaraskas) [inline docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc:241]: if you're using code formatting for the #, the formatting should match the yaml below. could also remove the code formatting

```suggestion
For example, with 25,000 running {agents} you could set the cache value to `50000`:
```

**Review by shainaraskas — APPROVED** [review]

**Review by juliaElastic — APPROVED** [review]
Looks great, thanks David.

**Review by karenzone — APPROVED** [review]
LGTM

**Pull request merged**